README.md

```diff
@@ -21,15 +21,26 @@ $ cd jukebox-js
 
 ## Development
 
-You can run everything in `Docker`. From the root of the repo, you just need to run:
+### Environment
+
+First, create a `.env` file in the root directory. Example vars can be found in the `.env.example` file. At a minimum `WS_MOPIDY_URL`, `WS_MOPIDY_PORT` and `CLIENT_ID` values need to be defined.
+
+`WS_MOPIDY_URL` can point either to a [local](docs/mopidy_install.md) instance of Mopidy or the production/staging instance in the office.
+
+`CLIENT_ID` can be set to anything for the app to launch, but needs to be set to a valid Google app ID in order for login to function.
+
```

Comment:
NOTE: You can't actually have any interaction with the app without logging in.

Reply:
👍 tweaked the wording.